types/src/front/content_fragment.ts

```diff
@@ -20,7 +20,6 @@ export type ContentFragmentContextType = {
 
 export const supportedContentFragmentType = [
   ...supportedUploadableContentType,
-  "dust-application/slack",
```

Comment:
`nit` but not sure `supportedContentFragmentType` brings a lot of value now, it's just a pass-through.

Reply:
fixed :+1:

Reply:
We need to keep it for backward compatibility until we backfill old content fragment I believe

Reply:
Yes will likely remove it but keeping it for now as we still have 2 content fragment flows